src/ADAL.PCL.iOS/TokenCachePlugin.cs

```diff
@@ -26,12 +29,66 @@ namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
     internal class TokenCachePlugin : ITokenCachePlugin
     {
+        private const string LocalSettingsContainerName = "ActiveDirectoryAuthenticationLibrary";
```

Comment:
I am not sure if I follow. The structure of cache in ADAL .NET and ADAL iOS (native) are different, so how do you expect the cache from one to work for another? Or I may be missing something here.